bootstrap.py

```diff
@@ -383,7 +383,10 @@ def install_puppet_agent():
     call_yum("install", "puppet")
     enable_service("puppet")
 
-    puppet_conf = open('/etc/puppet/puppet.conf', 'wb')
+    if os.path.isfile('/etc/puppet/puppet.conf'):
```

Comment:
can we turn that around and handle `/etc/puppetlabs` as the special case? feels slightly "cleaner" to me